differ/differ.go

```diff
@@ -134,6 +137,11 @@ const (
 	DefaultEventFilter         = "totalRisk >= totalRiskThreshold"
 )
 
+// Constants used for creating Service Log entries
+const (
+	serviceLogMaxTextLength = 255
```

Comment:
where does 255 comes from?

Reply:
there is a comment later:
```
	// It is necessary to truncate the fields because of Service Log limitations
```

Reply:
I observed that when I send messages with longer values, I receive an error from Service Log - should I add a comment on this to code?

Reply:
> there is a comment later:
> 
> ```
> 	// It is necessary to truncate the fields because of Service Log limitations
> ```

Yes, that's actually true. But should I mention it also near the constant declaration?